cypress/lib/helper.js

```diff
@@ -24,6 +12,16 @@ Profile ID: \
 Team member: ionut\
 Status ID:`
 
+export const bookingsList = []
+for (let i = 0; i <= 24; ++i) {
```

Comment:
can we have a multiple of 10 here or is it important to have exactly 25 bookings?

Reply:
We can have multiple of 10